guides/v2.2/cloud/project/project-webint-basic.md

```diff
@@ -80,6 +80,9 @@ One of these environment variables enables or disables outgoing emails for the e
 
 Configure your email notifications, services, and more as needed through the Magento Admin and test emails.
 
+{: .bs-callout-warning }
+If **Outgoing emails** option is equal to Off then [Unable to send mail: Unknown error]({{ site.baseurl }}/cloud/trouble/unable-send-email.html) will be displayed.
```

Comment:
To avoid using "will be" and make it read more like a real warning, consider changing the order...

```suggestion
You may receive an [Unable to send mail: Unknown error]({{ site.baseurl }}/cloud/trouble/unable-send-email.html) message if the **Outgoing emails** option is set to Off.
```

Reply:
Hi, @hguthrie!

**Done**